proxy/nginx/conf.d/ssl.conf

```diff
@@ -0,0 +1,5 @@
+ssl_session_cache shared:SSL:10m;
```

Comment:
@sakul-kaowvichit : Please do a qualsys SSL test or something similar. We can purchase a domain so you can do an end to end test? I don't think TLS 1.0 should be here